RedRock.Calendar.Modules.Users.Service/UserService.cs

```diff
@@ -28,8 +28,19 @@ public async Task<IEnumerable<UserDTO>> GetUsers()
         public async Task<UserDTO> GetUserById(Guid id)
         {
             var result = await userRepository.GetUserByIdAsync(id);
-            
+
             return result == null ? null : _mapper.Map<UserDTO>(result);
         }
+
+        public async Task<UserDTO> Login(string username, string password)
+        {
+            if (password.Length < 8) return null;
```

Comment:
do not use hardcoded number/string/other values, declare a variable instead (the naming can help to understand the value)